integration-cli/docker_cli_cp_to_container_test.go

```diff
@@ -35,7 +35,8 @@ func (s *DockerSuite) TestCpToErrSrcNotExists(c *check.C) {
 	c.Assert(os.IsNotExist(srcStatErr), checker.True)
 
 	err := runDockerCp(c, srcPath, dstPath, nil)
-	c.Assert(strings.ToLower(err.Error()), checker.Contains, strings.ToLower(srcStatErr.Error()))
+	c.Assert(os.IsNotExist(srcStatErr), checker.True)
+	c.Assert(strings.ToLower(err.Error()), checker.Contains, strings.ToLower(tmpDir))
```

Comment:
Lets see if this works instead @johnstep  - problem was that `os.Stat()` returns a different kind of error than `runDockerCp()`, so not sure if this is correct 😅